### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by jiri-janousek**

### CODE REVIEW

**Comment by jiri-janousek** [comment]
`test_anonymous_user_not_found (djangocms_oidc.tests.test_middleware.MiddlewareTestCase)` fails with Python 3.5 because this version doesn't maintain insertion order in dictionaries. Something like this could work:

```py
raise Exception(
  'Unable to authenticate with '
  + ', '.join(sorted('{!r}={!r}'.format(k, v) for k, v in credentials.items()))
)

raise Exception('Unable to authenticate with %r' % sorted(credentials.items()))
```

File: djangocms_oidc/constants.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 DJNAGOCMS_PLUGIN_SESSION_KEY = 'djangocms_oidc_plugin_id'
-DJNAGOCMS_USER_SESSION_KEY = 'dinagocms_oidc_user_info'
+DJNAGOCMS_USER_SESSION_KEY = 'djnagocms_oidc_user_info'
```
Comment (jiri-janousek) [inline djangocms_oidc/constants.py]: There is also a typo in `DJNAGOCMS_`.

Comment (zbohm) [reply djangocms_oidc/constants.py]: Typo fixed and an order in credentials too.

**Review by jiri-janousek — COMMENTED** [review]
Ready for merge.

**Pull request merged**